packages/headless/src/modules/order/services/customer.service.ts

```diff
@@ -33,45 +38,46 @@ export class OrderCustomerService {
     );
   }
 
-  async reOrder(userId: string, body: ReOrderQuery): Promise<IServiceResponse<CartResponse>>{
+  async reOrder(userId: string, body: ReOrderQuery): Promise<ReOrderResponse>{
     let { ignoreInvalidItems, overWriteCart, orderId } = body; 
 
     const prevOrder = await this.orderRepository.findOrder({orderId, userId});
-    if(!prevOrder) return errorResponse( ErrorMessageReOrder.INVALID_ID,null,HttpStatus.BAD_REQUEST );
+    if(!prevOrder) return {error: ErrorMessageReOrder.INVALID_ID,errors: null, code: HttpStatus.BAD_REQUEST};
```

Comment:
what was the problem here? i mean why this change is needed?

Reply:
as the user is not requesting for anything invalid initiaaly, so it shouldn't be a bad request or any kind of error response

Reply:
also the frontend team was unable to retrieve the error as it was getting caught in catch block